imports/wasi_snapshot_preview1/wasmedge.go

```diff
@@ -132,6 +132,11 @@ func (m *Module) WasmEdgeV2SockRecvFrom(ctx context.Context, fd Int32, iovecs Li
 }
 
 func (m *Module) WasmEdgeSockSetOpt(ctx context.Context, fd Int32, level Int32, option Int32, value Pointer[Int32], valueLen Int32) Errno {
+	// See socket.go
+	switch wasi.SocketOptionLevel(level) {
+	case wasi.TcpLevel:
+		option = option - 0x1000
```

Comment:
Should this be `option += 0x1000`?

Reply:
sorry just saw your comment. I wasn't done :) should be working now :)